attendance/attendance.py

```diff
@@ -7,13 +7,15 @@
 from urllib.request import urlopen
 from subprocess import Popen, PIPE
 from os.path import expanduser
+from os import getcwd
 from sys import platform as _platform
 
-
-if _platform == "linux" or _platform == "linux2":
+if "linux" in _platform:
     file_path = "/opt/attendance/"
 elif _platform == "darwin":
     file_path = f"/Users/{sys.argv[1]}/.attendance/"
+elif "win" in _platform:
+    file_path = getcwd()+'\\'
```

Comment:
Okay,
We should have a folder for storing the script right,
I would propose to have it in a folder itself like `.attendance` for having uniformity